.mergify.yml

```diff
@@ -75,11 +75,11 @@ pull_request_rules:
       merge:
         method: squash
         strict: smart+fasttrack
-  - name: delete upstream branch after merging changes on dev-tools/integration/.env
+  - name: delete upstream branch after merging changes on dev-tools/integration/.env or .go-version
```

Comment:
Use mergify to tidy up the branches that have merged with any of the existing automation we have put in place.